### PR TITLE
chore(plugintest): Convert Title to use v3 "for" store data.

### PR DESCRIPTION
**Pull request by BenJamesBen**

### CODE REVIEW

File: designs/plugintest/src/plugin-title.mjs [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { titlePlugin } from '@freesewing/plugin-title'
 import { base } from './base.mjs'
 
-const pluginTitle = ({ points, Point, paths, Path, macro, options, part }) => {
+const pluginTitle = ({ points, Point, paths, Path, macro, options, store, part }) => {
   if (['title', 'all'].indexOf(options.plugin) !== -1) {
-    if (options.titleMeta) part.context.settings.metadata = { for: 'Some user' }
+    if (options.titleMeta) store.data.for = 'Some user'
```
Comment (joostdecock) [inline designs/plugintest/src/plugin-title.mjs:6]: Can we use `store.set('data.for', 'Some user')` here?
It's the same under the hood today, but using the method is more future-proof

File: designs/plugintest/src/plugin-title.mjs [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+    else delete store.data.for
```
Comment (joostdecock) [inline designs/plugintest/src/plugin-title.mjs:7]: Same remark here, better to use `store.remove('data.for')`

**Review by joostdecock — CHANGES_REQUESTED** [review]

File: designs/plugintest/src/plugin-title.mjs (hunk 1 above)
Comment (BenJamesBen) [reply designs/plugintest/src/plugin-title.mjs:6]: I made the change to `store.set('data.for', 'Some user')`.

**Comment by BenJamesBen** [comment]
Hmm... my attempts to update the branch brought in too many other files. Closing and re-submitting.

**Pull request closed without merging**